README.md

```diff
@@ -58,22 +61,24 @@ that can be used as a dependency in any other wallet based project. It is develo
 ✔️: Successfully compiled  
 ❌: Not functioning, requires further attention
 
-## Notice
+⚠️ **Notice:**  
+We are fully using [Jetpack Compose](https://developer.android.com/jetpack/compose) and are now in the process of converting to support **Kotlin Multiplatform**.
 
-:warning: We are fully committed to implement [Jetpack Compose](https://developer.android.com/jetpack/compose) and moving ourself to support 
-`kotlin multi-platform`. **If you are sending any PR regarding `XML changes` we will `not` consider at this moment but converting XML to jetpack compose are most welcome.** If you sending any PR regarding logical changes in Activity/Fragment you are most welcome.
 
+Contributions are welcome in the following areas:
+- Development and enhancements related to Kotlin Multiplatform.
+- Logical changes or improvements in existing Jetpack Compose-based features.
+
+We appreciate your contributions and look forward to collaborating with you!
 
 | Development                                                                                                                                | Chat                                                                                                                                     |
 |--------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
 | ![Mobile-Wallet CI[Master/Dev]](https://github.com/openMF/mobile-wallet/workflows/Mobile-Wallet%20CI%5BMaster/Dev%5D/badge.svg?branch=dev) | [![Join the chat at https://mifos.slack.com/](https://img.shields.io/badge/Join%20Our%20Community-Slack-blue)](https://mifos.slack.com/) |
```

Comment:
update this badge with updated workflow file this is old and we don't have this workflow, and can you upload some badges like kotlin, kmp, etc to top of the file.